apps/website/next-sitemap.config.js

```diff
@@ -1,10 +1,13 @@
 // @ts-check
-/** @type {import('next-sitemap').IConfig} */
-module.exports = {
-  // If there is a NEXT_PUBLIC_VERCEL_URL set, use that like NextAuth.js does
-  siteUrl: process.env.NEXT_PUBLIC_VERCEL_URL
+
+const siteUrl =
+  (process.env.NEXT_PUBLIC_VERCEL_URLnpm
     ? `https://${process.env.NEXT_PUBLIC_VERCEL_URL}`
-    : process.env.NEXT_PUBLIC_BASE_URL,
+    : process.env.NEXT_PUBLIC_BASE_URL) || "http://localhost";
+
+/** @type {import('next-sitemap').IConfig} */
+const config = {
+  siteUrl: siteUrl,
```

Comment:
```suggestion
  // If there is a NEXT_PUBLIC_VERCEL_URL set, use that like NextAuth.js does
  // Fallback to localhost if we don't have a URL from the env for any reason
  siteUrl: (process.env.NEXT_PUBLIC_VERCEL_URL
    ? `https://${process.env.NEXT_PUBLIC_VERCEL_URL}`
    : process.env.NEXT_PUBLIC_BASE_URL) || "http://localhost",
```
We can just do this inline as before rather than having a constant